packages/shared/dropdown/multiselectdropdown.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+import { Select, SelectOption, SelectVariant } from '@patternfly/react-core';
+
+export const MultiSelectDropdown: React.FC<MultiSelectDropdownProps> = ({
+  onChange,
+  placeholder,
+  id,
+  options,
+  defaultSelected = [],
+}) => {
+  const [isOpen, setOpen] = React.useState(false);
+  const [selected, setSelected] = React.useState<string[]>(defaultSelected || []);
+  const onSelect = (event: React.MouseEvent | React.ChangeEvent, selection: string) => {
+    let cSelected: string[] = selected;
+    if (selected.includes(selection)) {
+      cSelected = selected.filter((item) => item !== selection);
+    } else {
+      cSelected = [...selected, selection];
+    }
+    setSelected(cSelected);
+    onChange(cSelected);
+  };
+
+  const { t } = useTranslation('plugin__odf-console');
+
+  const items: JSX.Element[] = options.map((item) => {
+    return <SelectOption key={item} value={item} />;
+  });
+  return (
+    <div>
```

Comment:
```suggestion
    <div>
```
Is this really required?

Reply:
not sure why exactly this was added... this is old component, I just copied it here...